### PR TITLE
LXC container monitoring

### PR DESCRIPTION
**Pull request by mfuhrmann**
It's a small use-case blog about snmp process monitor using the example of LXC container processes.

### CODE REVIEW

**Comment by mfuhrmann** [comment]
https://issues.opennms.org/browse/ORG-75

**Comment by mfuhrmann** [comment]
@indigo423 I have revised the wiki article and added the correct wiki link into the blog article.

File: user/pages/05.blog/2017-10-17-lxc-process-monitor/item.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+https://wiki.opennms.org/wiki/LXC_container_monitoring---
```
Comment (indigo423) [inline user/pages/05.blog/2017-10-17-lxc-process-monitor/item.md]: Is this a copy & paste error?

File: user/pages/05.blog/2017-10-17-lxc-process-monitor/item.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+title: Monitor LXC container processes using HostResourceSwRunMonitor
+date: 18:45 2017-11-27
+author: Marcel Fuhrmann
+body_classes: header-lite fullwidth blogstyling
+taxonomy:
+    category: wiki
+    tag: [use-case, configuration, snmp, wiki]
```
Comment (indigo423) [inline user/pages/05.blog/2017-10-17-lxc-process-monitor/item.md]: you can add "container" and "lxc"

File: user/pages/05.blog/2017-10-17-lxc-process-monitor/item.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+
+A monitoring solution should always help in bad situations and provide a useful overview of your system health status.
+Service availability monitors are especially handy to ensure that all services are running on the correct machines.
+
+In past, I have run into an problem which was more of an issue with documentation than a classical monitor alarming issue.
+We had some servers with LXC containers running.
```
Comment (indigo423) [inline user/pages/05.blog/2017-10-17-lxc-process-monitor/item.md]: LXC link to https://en.wikipedia.org/wiki/LXC

**Review by indigo423 — APPROVED** [review]

**Pull request merged**